discovery-infra/tests/test_e2e_install.py

```diff
@@ -4,25 +4,23 @@
 
 from tests.base_test import BaseTest
 from tests.config import ClusterConfig, TerraformConfig
-from tests.conftest import get_available_openshift_versions
+from tests.conftest import get_available_openshift_versions, get_api_client
 
 
 class TestInstall(BaseTest):
 
     @JunitTestSuite()
-    @pytest.mark.parametrize("openshift_version", get_available_openshift_versions())
+    @pytest.mark.parametrize("openshift_version", sorted(get_available_openshift_versions()))
     def test_install(self, get_nodes, get_cluster, openshift_version):
         new_cluster = get_cluster(cluster_config=ClusterConfig(openshift_version=openshift_version), nodes=get_nodes())
         new_cluster.prepare_for_installation()
         new_cluster.start_install_and_wait_for_installed()
 
     @JunitTestSuite()
-    # TODO: Fix OCS
-    # @pytest.mark.parametrize("olm_operator", get_api_client().get_supported_operators())
-    @pytest.mark.parametrize("olm_operator", ["lso", "cnv"])
-    def test_olm_operator(self, get_nodes, get_cluster, olm_operator):
-        new_cluster = get_cluster(cluster_config=ClusterConfig(olm_operators=[olm_operator]),
-                                  nodes=get_nodes(TerraformConfig(olm_operators=[olm_operator])))
+    @pytest.mark.parametrize("operators", sorted(get_api_client().get_supported_operators()))
```

Comment:
Why not using `api_client` fixture?

Reply:
No good reason :)

Reply:
I explored and see that using fixture in this place won't work: `Fixtures are meant to be called directly, but are created automatically`. We need to run this test in parallel for each operator.